Gui/Menu/DropDownList.cs

```diff
@@ -586,29 +579,29 @@ public MenuItem AddItem(string itemName, string itemValue = null)
 			return menuItem;
 		}
 
+		public int GutterWidth { get; set; } = 35;
+
 		public MenuItem AddItem(ImageBuffer leftImage, string itemName, string itemValue = null)
 		{
-			GuiWidget normalTextWithMargin = GetMenuContent(itemName, leftImage, MenuItemsBackgroundColor);
-			GuiWidget hoverTextWithMargin = GetMenuContent(itemName, leftImage, MenuItemsBackgroundHoverColor);
+			GuiWidget normalTextWithMargin = GetMenuContent(itemName, leftImage, MenuItemsBackgroundColor, MenuItemsTextColor);
+			GuiWidget hoverTextWithMargin = GetMenuContent(itemName, leftImage, MenuItemsBackgroundHoverColor, MenuItemsTextHoverColor);
 
-			MenuItem menuItem = new MenuItem(
+			var menuItem = new MenuItem(
 				new MenuItemStatesView(normalTextWithMargin, hoverTextWithMargin),
 				// If no itemValue is supplied, itemName will be used as the item value
-				itemValue ?? itemName);
-
-			// MenuItem is a long lived object that is added and removed to new containers whenever the
-			// menu is shown. To ensure that event registration is not duplicated, always remove before add
-			menuItem.Selected -= MenuItem_Clicked;
+				itemValue ?? itemName)
+			{
+				Name = itemName + " Menu Item",
+				Text = itemName
+			};
 			menuItem.Selected += MenuItem_Clicked;
```

Comment:
MenuItem might be long lived but in this context, we just created it, no need to -= the event listener